nilearn/input_data/nifti_labels_masker.py

```diff
@@ -19,16 +19,17 @@ class _ExtractionFunctor(object):
 
     func_name = 'nifti_labels_masker_extractor'
 
-    def __init__(self, _resampled_labels_img_, background_label):
+    def __init__(self, _resampled_labels_img_, background_label, func="mean"):
```

Comment:
Do not set default arg in private functions. They should always explicitly receive the args from the calling function.
Only user facing public functions should have default args.

Reply:
Thanks for the advice, I wasn't aware of that but it definitely makes sense. Fixed.